modules/manage/partials/tiered-storage/aws-kms-key.adoc

```diff
@@ -0,0 +1,67 @@
+[discrete]
+= Configure access to AWS S3 using an AWS KMS key
+
+When there are strict data compliance requirements and you must manage and store encryption keys separately from your cloud provider, you can configure an Amazon S3 bucket that Tiered Storage can use to leverage your customer-provided key (SSE-KMS) instead of the default AWS-managed key (SSE-S3).
+
+To convert an existing S3 bucket and its contents, you must:
+
+. Create a new KMS key
+. Configure the S3 bucket to use the new KMS key
+. (Optional) Re-encrypt existing objects to use the new KMS key
+
+[NOTE]
+====
+ifdef::env-cloud[]
+You cannot configure a cloud-provider managed encryption key at the topic level or in Redpanda Cloud Dedicated clusters.
```

Comment:
The Tiered Storage page isn't single-sourced into the Cloud docs. Not yet, at least.